modules/oadp-ibm-power-test-support.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc
+
+:_content-type: CONCEPT
+[id="oadp-ibm-power-test-matrix_{context}"]
+= OADP support for target backup locations using IBM Power Systems
+
+IBM Power Systems ("cloud systems") running on OCP 4.11 and OADP 1.1.0 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running IBM Power Systems with OCP 4.11 and OADP 1.1.0 against all non-AWS S3 backup location targets.
```

Comment:
@RichardHoch 
Why is it 'running on' and then 'running with'? IBM Power Systems is hardware, it doesn't run 'on' software. The opposite would be true - OCP runs on IBM Power Systems. So I would say 'running with' in both sentences. 
Also, I would add 'as well' at the end of the second sentence to make the point.